resources/assets/components/CampaignSingle/index.js

```diff
@@ -91,8 +91,10 @@ class CampaignSingle extends React.Component {
 
   render() {
     const posts = this.props.posts;
+    const postOrder = map(orderBy(posts, 'created_at', 'desc'), 'id');
```

Comment:
This is fantastically simple. 🎉

Reply:
wait I think I am still confused, why do we have to sort on the frontend again here?

Reply:
When we use `keyBy` here: https://github.com/DoSomething/rogue/blob/master/resources/assets/components/WithReviewing/index.js#L47
they get ordered by `post_id` ascending so we need to sort them again here in order to display them in the right order

Reply:
Can we create the two separate `posts` (keyed by ID) and `postIds` (ordered array) in that `WithReviewing` component? That'd also help make it more easily accessible other places where we want this behavior.

Reply:
@DFurnes would that be adding `postIds` to the state or making a method like `getPostOrder` to return `postIds`?

Reply:
I think you would just have to add `postsIds` to the state in `WithReviewing` the component it wraps should have access to `postIds` as a prop.

Reply:
Yep, what Shae said!